### ISSUE TITLE
CAN_NOT_GET_GATEWAY_SERVER

### ISSUE THREAD

**Issue by tsecret** [grishka/Houseclub#127]
**Describe the bug**
Not a bug from this project, but Agora related.

I am trying to connect to clubhouse rooms via Agora JS module (v4.3.0). However gettings this error when trying to connect to the any channel: 
`{name: "AgoraRTCException", code: "CAN_NOT_GET_GATEWAY_SERVER", message: "AgoraRTCError CAN_NOT_GET_GATEWAY_SERVER: no active status"}`

I have agora working in different projects, but having issues with clubhouse. I believe I am passing the right parameters to the functions. Is there anything else I need to do before except for creating Agora client and simply joining the channel?

**Comment by grishka** [comment]
I've never seen this error. The only parameters it needs are the API key, the channel token, the channel "name" (the alphanumeric ID), and the current user ID. These two calls were all I needed to get it working.

https://github.com/grishka/Houseclub/blob/08e05311503ced095c06b86bec5fe34039b82094/Houseclub/src/main/java/me/grishka/houseclub/VoiceService.java#L90
https://github.com/grishka/Houseclub/blob/08e05311503ced095c06b86bec5fe34039b82094/Houseclub/src/main/java/me/grishka/houseclub/VoiceService.java#L149

**Comment by NothingEverHappens** [comment]
Hey @TSecretT, were you able to figure it out?
Would you like to sync and try digging into it together?

**Comment by tsecret** [comment]
> Hey @TSecretT, were you able to figure it out?
> Would you like to sync and try digging into it together?

Nope, I have tried to connect to agora demo call via the project and it worked, as soon as I enter clubhouse params I get this error. I don't really think so, but clubhouse auth/refresh tokens have nothing to do with agora connections, right?

**Comment by NothingEverHappens** [comment]
we're debugging it right now, can you join https://www.twitch.tv/kirjs if you have some time?
(I'll remove the link in a few minutes)

**Comment by lafin** [comment]
@TSecretT since yesterday, web transport to the agora seems to be off by the clubhouse. worked great a few days before. I think this is related to this news https://www.bloomberg.com/news/articles/2021-02-22/clubhouse-chats-are-breached-raising-concerns-over-security

**Comment by r0manchak** [comment]
@TSecretT Hi, do you have any progress with this error?
I have the same one and no idea how to overcome

**Comment by devbymak** [comment]
I still have not found a solution. I am tired of trying

**Comment by Sotatek-ThongLe** [comment]
Is there any solutions for this? My project worked fine few days ago and today I get this error.

**Comment by sooxt98** [comment]
I believe that Agora has disabled clubhouse webrtc support, and only allow connections from the native's SDK, I just tried with agora flutter sdk it works just fine and it support listening to multiple room simultaneously too, but you have to call `active_ping` constantly to avoid audio drop (update: it turns out that something is dropping the agora audio after few mins after i killed the app)

Here's others issue pointing to same problem
https://github.com/ai-eks/OpenClubhouse/issues/13
https://github.com/tomaspavlin/clubhouse-client/issues/3
